docs/index.rst

```diff
@@ -5,7 +5,7 @@ pep8's documentation
 
 .. note::
 
-    This package used to be called ``pep8`` but was renamed to ``pycodestyle``
+    This package used to be called `pep8 <https://www.python.org/dev/peps/pep-0008/>`_ but was renamed to `pycodestyle <https://pypi.python.org/pypi/pep8>`_
```

Comment:
There's a few things wrong with this:

1. There was a maximum line length being observed in this document that is being violated by this line
2. The first link should not be to the document PEP-0008, but instead to the package on PyPI pep8.
3. The second link should not be to the package on PyPI named "pep8" but to the one named "pycodestyle"